presentation/src/main/java/com/exallium/mvvmcleanexample/presentation/utils/PropertyCacher.kt

```diff
@@ -0,0 +1,37 @@
+package com.exallium.mvvmcleanexample.presentation.utils
+
+import kotlin.reflect.KProperty
+
+object PropertyCacher {
```

Comment:
It is boilerplate, but it avoids having to reference BR.propertyName for each changed property in an Observable object (Observable from the data binding framework I mean).